<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationSelsetDog</class>
 <widget class="QDialog" name="SimulationSelsetDog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-3</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnSelectOk">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择目录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>图片目录</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Import">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ImageImport</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Save">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ImageSave</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Failed">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FailedImage</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_OK">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>OKImage</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Base">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BaseImage</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
